client/app/students/new-student/new-student.html

```diff
@@ -5,6 +5,7 @@
         </div>
     </md-toolbar>
     <form name="addStudentForm">
+       <div>
```

Comment:
Why did you add this empty div? it doesn't seem to do anything
Delete it if it's unnecessary